### ISSUE TITLE
Back/Forward and Refresh/Stop on toolbar

### ISSUE THREAD

**Issue by K4sum1** [fork-maintainers/iceraven-browser#675]
I don't like having to go into a menu to use basic functionality of the browser. There should be an option to add these buttons to the toolbar like it was in Fennec.

**Comment by akliuxingyuan** [comment]
I don't think it's a good idea. Not like Firefox desktop, toolbar don't have enough space to display other things

**Comment by Arkuarius** [comment]
> I don't think it's a good idea. Not like Firefox desktop, toolbar don't have enough space to display other things

@akliuxingyuan eh? I mean look at Safari on iOS, they have the forward, backward, and refresh button right under the address bar. Would it be impossible to rewrite Firefox' address bar to be something like that? 
![IMG_20231112_154316](https://github.com/fork-maintainers/iceraven-browser/assets/102014881/0cf3f4ef-5f13-49ac-933d-a9aae89f213a)

**Comment by akliuxingyuan** [comment]
I thought you want put all buttons next to the url..
a separate row of space is much better, but I don't know if it can be implemented, need some time to research the code

**Comment by K4sum1** [comment]
> I thought you want put all buttons next to the url..

That's how it was in Fennec, and it worked for me.
![ScreenShot_20231112091146](https://github.com/fork-maintainers/iceraven-browser/assets/77039240/12f706a8-d03f-4a74-9517-1bee382b0dd2)

**Comment by Arkuarius** [comment]
> > I thought you want put all buttons next to the url..
> 
> That's how it was in Fennec, and it worked for me. ![ScreenShot_20231112091146](https://user-images.githubusercontent.com/77039240/282307625-12f706a8-d03f-4a74-9517-1bee382b0dd2.png)

YES this is exactly how I want a future version of Iceraven to look like!
